web/files/files.go

```diff
@@ -1574,6 +1577,9 @@ func FindFilesMango(c echo.Context) error {
 		} else {
 			if ok {
```

Comment:
I tried but it's harder than it looks because `newFindFile` and `NewFile` returns different types. And using an interface is quite hard as there are other places where we use methods only implemented by one of those types.

Reply:
I see. Well, let's keep this refactoring for later then.